core/platform/email/gae_email_services.py

```diff
@@ -22,6 +22,11 @@
 
 
 def get_incoming_email_address(reply_to_id):
+    """Gets the incoming email address. The client is responsible for recording any audit logs.
+       
+    Args:
+      - reply_to_id: str. registered username of sender.
+    """
```

Comment:
Remove hyphen(-) at the beginning. You can do it for all other places also.
reply_to_id is the unique 'id' of the sender. PTAL! carefully and check.